go.mod

```diff
@@ -179,6 +180,7 @@ require (
 	github.com/google/go-querystring v1.0.0 // indirect
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/google/pprof v0.0.0-20211214055906-6f57359322fd // indirect
+	github.com/google/renameio v0.1.0 // indirect
```

Comment:
I wonder where v0 is coming from...

Reply:
What happens when you run `go mod tidy`?

Reply:
Did the trick!